spec/requests/order_items_spec.rb

```diff
@@ -13,4 +19,23 @@
       expect(JSON.parse(response.body)['data'].first['id']).to eq(order_item.id.to_s)
     end
   end
+
+  describe "#approval_requests" do
+    let!(:approval) { create(:approval_request, :order_item_id => order_item.id, :workflow_ref => "1") }
+
+    context "list" do
+      before do
+        get "#{api}/order_items/#{order_item.id}/approval_requests"
```

Comment:
@lindgrenj6 add :headers => default_headers

Reply:
that actually fixed it.